### DISCUSSION TITLE
京东商品页面直播小窗无法定位

### DISCUSSION THREAD

**Discussion by superopfan** [gkd-kit/gkd#845] [Q&A] [answered]
请问京东商品页面的直播小窗有没有办法可以去掉, 我在快照里面怎么也定位不到这个悬浮窗的元素
![QQ_1735552453288](https://github.com/user-attachments/assets/671e9d0e-552d-4562-9f04-eb16a32e0035)

**Comment by lisonge** [comment]
需要给出快照

多给点信息吧，光看也猜不出来啊

**Reply by superopfan** [reply]
不好意思,之前没有上传快照,现在上传了,麻烦看下怎么定位
[京东_ProductDetailActivity-1735779909424.zip](https://github.com/user-attachments/files/18287128/_ProductDetailActivity-1735779909424.zip)

**Reply by lisonge** [reply, marked as answer]
<https://i.gkd.li/i/18287438>

说明京东没有提供这个小窗的有效无障碍信息，跟 gkd 无关

**Reply by superopfan** [reply]
感觉解答,我还以为和html一样每个节点都能定位出来模拟点击

**Reply by lisonge** [reply]
应用可以选择不提供无障碍信息，比如货拉拉，整个界面都没有无障碍信息
